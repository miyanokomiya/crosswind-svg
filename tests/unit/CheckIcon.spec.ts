import { mount } from "@vue/test-utils";
import Target from "../../src/components/CheckIcon.vue";

describe("CheckIcon", () => {
  it("checked:false", () => {
    const wrapper = mount(Target, {
      propsData: { checked: false }
    });
    expect(wrapper.element).toMatchSnapshot();
  });
  it("checked:true", () => {
    const wrapper = mount(Target, {
      propsData: { checked: true }
    });
    expect(wrapper.element).toMatchSnapshot();
  });
});
