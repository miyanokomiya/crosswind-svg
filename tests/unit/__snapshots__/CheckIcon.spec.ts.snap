// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CheckIcon checked:false 1`] = `
<svg
  style="transform: rotate(0); fill: currentColor; stroke: currentColor; user-select: none;"
  viewBox="0 0 64 64"
  xmlns="http://www.w3.org/2000/svg"
  xmlns:xlink="http://www.w3.org/1999/xlink"
>
  <g
    fill="white"
  >
    <circle
      cx="32"
      cy="32"
      r="32"
    />
  </g>
   
  <g
    fill="none"
    stroke=""
    stroke-linecap="round"
    stroke-width="6"
  >
    <path
      d="M 16 32 28 44 50 22"
      style="transform-origin: 50% 50%; transition: all 0.5s; transform: scale(0);"
    />
  </g>
</svg>
`;

exports[`CheckIcon checked:true 1`] = `
<svg
  style="transform: rotate(0); fill: currentColor; stroke: currentColor; user-select: none;"
  viewBox="0 0 64 64"
  xmlns="http://www.w3.org/2000/svg"
  xmlns:xlink="http://www.w3.org/1999/xlink"
>
  <g
    fill="white"
  >
    <circle
      cx="32"
      cy="32"
      r="32"
    />
  </g>
   
  <g
    fill="none"
    stroke=""
    stroke-linecap="round"
    stroke-width="6"
  >
    <path
      d="M 16 32 28 44 50 22"
      style="transform-origin: 50% 50%; transition: all 0.5s; transform: scale(1);"
    />
  </g>
</svg>
`;
